test/autorest/generated/complexgroup/basic.go

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package complexgroup
```

Comment:
This file didnt change, I only renamed it since it was previously called complex.go which was too broad for the complex package since there are several different models within it.

Reply:
In track 1 we named the file to match the client name minus the `Client` suffix; let's do the same thing in track 2.

Reply:
Perfect that change is also part of this PR